benchmark/utils/cuda_linops.hpp

```diff
@@ -709,10 +710,10 @@ using cusp_gcoo = detail::CuspGenericCoo<>;
 
 #if defined(CUDA_VERSION) && (CUDA_VERSION < 11000)
 using cusp_coo =
-    detail::CuspHybrid<double, gko::int32, CUSPARSE_HYB_PARTITION_USER, 0>;
+    detail::CuspHybrid<etype, gko::int32, CUSPARSE_HYB_PARTITION_USER, 0>;
 using cusp_ell =
-    detail::CuspHybrid<double, gko::int32, CUSPARSE_HYB_PARTITION_MAX, 0>;
-using cusp_hybrid = detail::CuspHybrid<>;
+    detail::CuspHybrid<etype, gko::int32, CUSPARSE_HYB_PARTITION_MAX, 0>;
```

Comment:
For both here and HIP, we are probably not going to have all valuetypes work right? Should we throw an exception when that's the case, if some, where would be the most appropriate?

Reply:
I think it is easier to rely on the `GKO_NOT_SUPPORTED`, so we don't have to update it in multiple places.

Reply:
@tcojean if you are ok with this decision, I guess we can resolve this discussion and the pull request is ready to be merged.